### DISCUSSION TITLE
Port meow to Intellij?

### DISCUSSION THREAD

**Discussion by dromoli** [meow-edit/meow#334] [Ideas]
Hi @DogLooksGood, 

I am a heavy user of Intellij. Plugins like IdeaVim bring modal editing to Intellij, but meow is certainly an improvement over Vim-style keybindings.

I was wondering what would be the most straightforward way of porting meow to Intellij.

Have you ever considered this?

**Comment by DogLooksGood** [comment]
Well, I don't have this in my mind, since I'm not an IntelliJ user. Porting meow could be very time consuming, there's no straightforward way I guess.
